packages/ove-service-auth/README.md

```diff
@@ -0,0 +1,191 @@
+# OVE Base Proxy
+
+This is a generic base proxy that supports various configurations, provided either via environment variables or 
```

Comment:
What exactly do you mean by 'base' here?

Reply:
That this is a base service which can be used to create other services, see the scheduler service template that I've just added.